### PR TITLE
Api v5

### PR DESCRIPTION
**Pull request by abramsymons**

### CODE REVIEW

**Comment by adamstallard** [comment]
> I don't think we should add the library `'fast-json-stable-stringify'` for stringifying these simple objects, why is this library needed?

@RnbWd The reason to stringify with this library is to produce a deterministic (i.e. stable, same result every time) string. This way when the backend constructs the string message to check the signature, it will be constructed in exactly the same way.

**Comment by RnbWd** [comment]
@adamstallard ah okay that makes sense

File: BrightID/src/components/Apps/AppsScreen.js [hunk 1]
```diff
@@ -3,26 +3,20 @@
...
 import { ORANGE } from '@/utils/constants';
 import AppCard from './AppCard';
-import { handleAppContext, deleteApp } from './model';
-
-let deleteSheetRef = '';
-
-type State = {
-  selectedApp: string,
-};
+import emitter from '@/emitter';
```
Comment (TripleSpeeder) [inline BrightID/src/components/Apps/AppsScreen.js:10]: emitter, store and api are unused.

File: BrightID/src/components/Apps/model.js [hunk 2]
```diff
@@ -2,76 +2,46 @@
 import { Alert } from 'react-native';
 import api from '@/api/brightId';
 import { saveImage } from '@/utils/filesystem';
```
Comment (TripleSpeeder) [inline BrightID/src/components/Apps/model.js:4]: unused import

File: BrightID/src/components/Apps/model.js [hunk 3]
```diff
@@ -2,76 +2,46 @@
...
-import { addApp, removeApp } from '@/actions';
+import { addLink } from '@/actions';
 import { navigate } from '@/NavigationService';
 import store from '@/store';
-import { find, propEq } from 'ramda';
+import emitter from '@/emitter';
```
Comment (TripleSpeeder) [inline BrightID/src/components/Apps/model.js:8]: unused import

**Comment by TripleSpeeder** [comment]
Added some minor findings. Branch needs to be rebased.

File: BrightID/src/reducer/apps.js [hunk 4]
```diff
@@ -20,9 +21,23 @@ export const reducer = (state: AppsState = initialState, action: action) => {
       };
     }
     case SET_APPS: {
+      const contexts = action.apps.map((app) => app.context);
+      const linkedContexts = state.linkedContexts.filter(({ context }) =>
+        contexts.includes(context),
+      );
+      let oldLinkedContexts;
+      if (linkedContexts.length !== state.linkedContexts.lenght) {
```
Comment (RnbWd) [inline BrightID/src/reducer/apps.js:42]: typo

**Comment by RnbWd** [comment]
This branch updates the apps screen, displays a list of apps automatically, deletes all apps for the current user, updates the apps reducer, and updates the app to use v5 of the API

**Pull request merged**